qa/rpc-tests/feature_fedpeg.py

```diff
@@ -47,9 +49,30 @@ def __init__(self):
         self.setup_clean_chain = True
         self.num_nodes = 4
 
-    def setup_network(self, split=False):
+    def add_options(self, parser):
+        parser.add_option("--bitcoinddir", dest="bitcoinddir", default="",
+                          help="Use the bitcoind binary in the given directory")
 
-        # Parent chain args
+    def setup_parents_bitcoin(self):
+        self.extra_args = [[
+            "-regtest=1",
+            "-port="+str(p2p_port(n)),
+            "-rpcuser="+rpc_auth_pair(n)[0],
+            "-rpcpassword="+rpc_auth_pair(n)[1],
+            "-rpcport="+str(rpc_port(n)),
+            "-testnet=0",
+            "-txindex=1",
+            "-addresstype=legacy", # To make sure bitcoind gives back p2pkh no matter version
+            "-connect=localhost:"+str(p2p_port((n+1)%2)),
+            "-listen=1",
+            #"-debug"
+        ] for n in range(2)]
+
+        binary = os.path.join(self.options.bitcoinddir, "bitcoind")
+        self.nodes = start_nodes(2, self.options.tmpdir, self.extra_args[:2], binary=binary, chain="regtest")
```

Comment:
there's still no -chain=regtest option in normal bitcoind binaries, so in https://github.com/ElementsProject/elements/blob/elements-0.14.1/qa/rpc-tests/test_framework/util.py#L343 you will need to replace '-chain='+chain with just '-regtest' if chain == 'regtest'

Reply:
Yeah I wanted to try. Is this why it fails? I thought it might just ignore it.